### PR TITLE
Add Styvio

### PR DESCRIPTION
**Pull request by jaredcassoutt**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by jaredcassoutt** [comment]
Added the Styvio API to the Finance section of the repository's README.md file

File: README.md [hunk 1]
```diff
@@ -416,6 +416,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Plaid](https://plaid.com/) | Connect with users’ bank accounts and access transaction data | `apiKey` | Yes | Unknown |
 | [Razorpay IFSC](https://ifsc.razorpay.com/) | Indian Financial Systems Code (Bank Branch Codes) | No | Yes | Unknown |
 | [Real Time Finance](https://github.com/Real-time-finance/finance-websocket-API/) | Websocket API to access realtime stock data | `apiKey` | No | Unknown |
+| [Styvio](https://www.Styvio.com) | Realtime and historical stock data and current stock sentiment | No | No | Unknown |
```
Comment (matheusfelipeog) [inline README.md:419]: This API support HTTPS protocol

```suggestion
| [Styvio](https://www.Styvio.com) | Realtime and historical stock data and current stock sentiment | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution, we are grateful for that. 😄 

Can you resolve the review, please?

**Review by matheusfelipeog — APPROVED** [review]
Thanks for your update @jaredcassoutt, we are grateful for that ❤️

**Review by matias904 — APPROVED** [review]

**Review by matias904 — APPROVED** [review]

**Pull request merged**